02-data_management_plan.Rmd

```diff
@@ -9,7 +9,19 @@ ottrpal::set_knitr_image_path()
 # Data Management/Storage Options
 
 ## Data Storage Concerns
-How you should store and interact with data while doing research.  
+How you should store and interact with data while doing research. 
+
+Your data storage needs will depend greatly on the type of data you are storing as well as the number of samples. File sizes vary considerably based on the data being stored. A single file can be as small as 1 MB in size (for a PET scan image of the heart) to 60 GB (for an uncompressed fastq of a whole genome sequence). Storing files in a compressed format, especially raw files, can help decrease your storage needs and costs. Below is a table of common types of data and sizes for single files. This list is not comprehensive but instead should be taken as a general guide. You should always get a more specific estimate for your particular project before submitting your grant proposal.
+
+| **Type of data** | **Common size for a _single_ file** |
+|------------------|------------------|
+| Genomics (WGS, WES) |	15-60 GB |
+| Genomics (RNA-seq, scRNA-seq)| 3-25 GB |
+| Imaging (microscopy) | 2-8 MB |
+| Imaging (human medical) | 1 MB – 2.5 GB |
+| Flow cytometry |	
```

Comment:
can we source some average `fcs` file sizes? I can post a message in the slack.

Reply:
just did this, we'll see if we get any bites :)

Reply:
We can merge for now, not sure when we will get a good estimate here. In my book, something is better than nothing :)

Reply:
@avahoffman sounds good! I am looking through the files Sandy put together and am wondering if we should use the big table the Genomics core created somewhere.

Reply:
Looked at this article : https://pubmed.ncbi.nlm.nih.gov/32881398/ and didn't see much to estimate individual file size without diving deeper. I know we have experts at hutch.. just gotta pick their brains a bit 😜

Reply:
Yeah, I looked through a bunch of articles and resources this afternoon and couldn't find anything concrete. Based on the format of the files, I think it's probably on the order of MB, not GB, but that's still a huge range.

Reply:
That's cool! Good detective work :)